src/pages/SearchPage.js

```diff
@@ -79,20 +78,25 @@ class SearchPage extends Component {
             props.betas,
         );
 
-        this.state = {
-            searchValue: '',
-            recentReports,
-            personalDetails,
-            userToInvite,
-        };
+        this.recentReports = recentReports;
+        this.personalDetails = personalDetails;
+        this.userToInvite = userToInvite;
```

Comment:
Won't these need to be left in state so this can re-render with changes to these values?

Reply:
I didn't know that the SearchPage needs to re-render when updating props, but just found Onyx component wrapper.
Ok, I will change that to use this.props. (recentReports, personDetails, userToInvite)